src/lib/util/numbers.mli

```diff
@@ -155,4 +155,11 @@ module Q : sig
   val root_excess  : t -> int -> t option
   val sqrt_default : t -> t option
   val sqrt_excess  : t -> t option
+
+  val (~$$) : Z.t -> t
```

Comment:
Unrelated to the PR but really we should just get rid of the `Numbers` module and use `Z` and `Q` directly, this is a relic of the time we supported either Num or Zarith.

(Well there is the issue of `Q.to_Z` not asserting in Zarith I guess)

Reply:
I removed this code and remove the `module Q = numbers.Q` in `intervals`. We'll remove completely the module numbers later.

Reply:
Note, this may be dangerous — I am not sure that some functions such as `Q.floor` or `Q.ceiling` have the same semantics in `Q` and `Numbers.Q`; I'd prefer we did this in a seprate PR.

Reply:
... no